docs/user-guide/configuring-containers.md

```diff
@@ -0,0 +1,131 @@
+# Kubernetes User Guide: Managing Applications: Configuring and launching containers
+
+## Configuration in Kubernetes
+
+In addition to the imperative-style commands described [elsewhere](quick-start.md), Kubernetes supports declarative YAML or JSON configuration files. Often times, configuration files are preferable to imperative commands, since they can be checked into version control and changes to the files can be code reviewed, which is especially important for more complex configurations, producing a more robust, reliable and archival system.
```

Comment:
Maybe give examples to contrast what you mean by imperative vs declarative.

*In addition to the imperative-style commands, such as `kubectl run` and `kubectl expose`, described elsewhere, Kubernetes supports declarative configuration.  In the declarative style, all configuration is stored in YAML or JSON configuration files which are synced to the cluster via the `kubectl update` command.*